packages/utils/src/index.js

```diff
@@ -266,3 +266,31 @@ export function getWordCount(editorState: EditorState): number {
   const wordArray = cleanString.match(/\S+/g) // matches words according to whitespace
   return wordArray ? wordArray.length : 0
 }
+
+export function mergeEntityData(
+  editorState: EditorState,
+  entityKey: string,
+  data: Object
+): EditorState {
+  const newContentState = editorState
+    .getCurrentContent()
+    .mergeEntityData(entityKey, data)
+
+  return EditorState.push(editorState, newContentState, 'apply-entity')
+}
+
+export function mergeBlockData(
```

Comment:
Rather than adding a new utility method let's use this [draft js modifier](https://draftjs.org/docs/api-reference-modifier#mergeblockdata) in the example